src/Libraries/Microsoft.Extensions.AI.Abstractions/Utilities/AIJsonUtilities.Schema.cs

```diff
@@ -25,100 +25,90 @@ namespace Microsoft.Extensions.AI;
 /// <summary>Provides a collection of utility methods for marshalling JSON data.</summary>
 public static partial class AIJsonUtilities
 {
+    private const string SchemaPropertyName = "$schema";
+    private const string TitlePropertyName = "title";
+    private const string DescriptionPropertyName = "description";
+    private const string NotPropertyName = "not";
+    private const string TypePropertyName = "type";
+    private const string PatternPropertyName = "pattern";
+    private const string EnumPropertyName = "enum";
+    private const string PropertiesPropertyName = "properties";
+    private const string RequiredPropertyName = "required";
+    private const string AdditionalPropertiesPropertyName = "additionalProperties";
+    private const string DefaultPropertyName = "default";
+    private const string RefPropertyName = "$ref";
+
     /// <summary>The uri used when populating the $schema keyword in inferred schemas.</summary>
     private const string SchemaKeywordUri = "https://json-schema.org/draft/2020-12/schema";
 
-    /// <summary>Soft limit for how many items should be stored in the dictionaries in <see cref="_schemaCaches"/>.</summary>
```

Comment:
Note that this change removes the JSON schema caching since there's no longer a need for leaf clients to infer the schema on their side -- AIFunctions should always supply a valid schema now.